src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: Improving Core Web Vitals on the Mail.ru home page resulted in an average 10% increase in conversion rates
+subhead: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+description: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+authors:
+  - dstasyev
+  - svenmay
+date: 2021-11-15
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/D3aV6VLiccC31BJIwo3z.jpg
+alt: Google Core Web Vitals + Mail.ru Home Page.
+thumbnail: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/CWMTBHzJ9UCpZ89fNcxJ.jpg
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+{% Aside %}
+  This case study was authored by Denis Stasyev from the Mail.ru infrastructure team.
+{% endAside %}
+ 
+## Where we started
+ 
+Mail.ru is one of the leading e-mail services on the Russian-speaking Internet and is [in the top 5 sites in Russia in terms of traffic](https://www.alexa.com/topsites/countries/RU). The Mail.ru home page is an important resource for a large number of people. It serves over several hundred million visits per month and is a portal from where people can access email, news, social media, performance internet searches and more.
+ 
+Mail.ru wanted to provide its visitors with a high quality user experience, so work began to improve CWV. Before discussing Mail.ru's CWV optimizations, some technical details of the Mail.ru home page should first be noted.
+ 
+Though the project had been developed using our in-house templating engine [Fest](https://github.com/mailru/fest) over a long time, switched to [Svelte 3](https://svelte.dev/) in 2019. You can find out more about Fest and the reasons for changing the technology stack to Svelte in [our team's presentation](https://youtu.be/H2l3pK42f38?t=2913) at Svelte Russia Meetup #1.
+ 
+Svelte implements reactivity in a way that [doesn't use Virtual DOM](https://svelte.dev/blog/virtual-dom-is-pure-overhead), which makes it less resource-intensive. [Svelte's approach removes unused functions from production bundles](https://svelte.dev/blog/frameworks-without-the-framework) because the code implementing them isn't generated by the compiler if the functions aren't used. Unused code is removed during compilation, resulting in smaller bundles.
```

Comment:
```suggestion
Svelte implements reactivity in a way that [doesn't use Virtual DOM](https://svelte.dev/blog/virtual-dom-is-pure-overhead), which makes it less resource-intensive. [Svelte's approach removes unused functions from production bundles](https://svelte.dev/blog/frameworks-without-the-framework) because the code implementing them isn't generated by the compiler if the functions aren't used. Unused code is removed during compilation, resulting in smaller bundles. This may help reduce [Total Blocking Time (TBT)](https://web.dev/tbt/) during page startup.
```